internal/plugin/connectors/http/template/plugin.go

```diff
@@ -0,0 +1,33 @@
+package template
+
+import (
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector/http"
+)
+
+// PluginInfo is required as part of the Secretless plugin spec. It provides
+// important metadata about the plugin.
+func PluginInfo() map[string]string {
+	// TODO: fill in the map according to
+	// https://github.com/cyberark/secretless-broker/blob/master/pkg/secretless/plugin/connector/README.md#plugininfo
+	return map[string]string{
+		"pluginAPIVersion": "",
+		"type":             "",
```

Comment:
Type prefix for http plugins is standardized: https://github.com/cyberark/secretless-broker/blob/master/internal/plugin/connectors/http/basicauth/plugin.go#L13